src/librustc/hir/mod.rs

```diff
@@ -2014,7 +2014,7 @@ pub struct EnumDef {
 
 #[derive(Clone, RustcEncodable, RustcDecodable, Debug)]
 pub struct VariantKind {
-    pub name: Name,
+    pub ident: Ident,
```

Comment:
Please, move all the `Ident`ification into a separate commit, it's a large change not directly related to variants as associated items.

Reply:
Sure, will do.